TiledLib/TmxMisc.cs

```diff
@@ -60,6 +60,7 @@ public static void ReadTileset(this XmlReader reader, Tileset ts)
                         reader.ReadTile(ts.TileProperties, ts.TileAnimations);
                         break;
                     default:
+                        reader.Skip();
```

Comment:
This was causing loads to hang when encountering unexpected elements, such as the terrain element.